### PR TITLE
Susan/useradminui

### PR DESCRIPTION
**Pull request by sadelokiki**
Create UI Layout and validations for Login

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,7 +4,14 @@
   resources :backgrounds
   resources :reports
   resources :pipeline_outputs, only: [:index, :show]
-  devise_for :users
+  devise_for :users, controllers: {
+    sessions: 'sessions',
+    registrations: 'registrations'
+  }
+  devise_scope :user do
```
Comment (yunfangjuan) [inline config/routes.rb]: why do we need this? 
why not just use /users/sign_in, /users/sign_out?

File: app/assets/javascripts/components/login.jsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    this.setState.isChecked = !this.setState.isChecked;  
+  }
+
+  renderLogin() {
+    return (
+        <div className="form-wrapper">
+          <div className="row">
+            <form ref="form" className="new_user" id="new_user" onSubmit={ this.handleSubmit }>
+              <div className="row title">
+                <p className="col s6 verify">Not a user? Sign up</p>
```
Comment (yunfangjuan) [inline app/assets/javascripts/components/login.jsx]: We don't allow people to sign up directly so remove this line for now.

File: app/assets/javascripts/components/login.jsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                  <i className="fa fa-key" aria-hidden="true"></i>
+                  <input ref= "password" type="password" className="" onFocus={ this.clearError }   />
+                  <label htmlFor="user_password">Password</label>
+                </div>
+                <div className="">  
+                  <input ref="remember_me" type="checkbox" name="switch" className="filled-in" id="remember_me" onChange={ this.toggleCheckBox } value={this.setState.isChecked ? 1 : 0} />
+                  <label htmlFor="remember_me">Remember me</label>
+                </div>
+              </div>
+                <div className="forgot-password"><span onClick={ this.gotToForgotPassword }>Forgot your password?</span><br /></div>
```
Comment (yunfangjuan) [inline app/assets/javascripts/components/login.jsx]: the forget password flow seems a bit odd. probably not for this pr, but we should fix it soon.

File: config/routes.rb (hunk 1 above)
Comment (sadelokiki) [reply config/routes.rb]: I figured it is easier go to login, instead of the longer route? @yunfangjuan. It renders the same view.

Comment (yunfangjuan) [reply config/routes.rb]: but it's redudant and makes code management more complicated.

**Pull request merged**